packages/backend/migration/1688647797135-userannouncement.js

```diff
@@ -3,9 +3,11 @@ export class Userannouncement1688647797135 {
 
     async up(queryRunner) {
         await queryRunner.query(`ALTER TABLE "announcement" ADD COLUMN "userId" character varying(32)`);
+        await queryRunner.query(`ALTER TABLE "announcement" ADD COLUMN "closeDuration" integer`);
```

Comment:
nullable: false なので
```suggestion
        await queryRunner.query(`ALTER TABLE "announcement" ADD COLUMN "closeDuration" integer NOT NULL DEFAULT 0`);
```
とかつけないとだめかも